engine/access/rpc/backend/accounts/handler/handler.go

```diff
@@ -0,0 +1,16 @@
+package handler
+
+import (
+	"context"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type Handler interface {
+	GetAccountAtBlockHeight(ctx context.Context, address flow.Address, blockID flow.Identifier, height uint64) (*flow.Account, error)
```

Comment:
since these methods require both the blockID and height, maybe just call it `GetAccountAtBlock`